packages/node_modules/glimmer-test-helpers/lib/environment.ts

```diff
@@ -403,8 +403,24 @@ interface BasicComponentFactory {
 }
 
 abstract class GenericComponentDefinition<T extends Component> extends ComponentDefinition<T> {
-  compile({ layout }: { layout: CompiledLayout }): ComponentParts {
-    let { program } = layout;
+  public compiledLayout : CompiledLayout;
+  public layout : string;
+
+  constructor(name: string, manager: ComponentManager<any>, ComponentClass: ComponentClass, layout) {
+    super(name, manager, ComponentClass);
+    this.layout = layout;
+  }
+
+  getLayout(env) {
```

Comment:
Does this not have to be on the interface/abstract class? (If we call it outside of the test-helper file, then it probably needs to be on the abstract class)